provider/resources.go

```diff
@@ -219,5 +221,32 @@ func Provider() tfbridge.ProviderInfo {
 	return prov
 }
 
+func docEditRules(defaults []tfbridge.DocsEdit) []tfbridge.DocsEdit {
+	return append(defaults, tfbridge.DocsEdit{
+		Path: "service.html.markdown",
+		Edit: func(_ string, content []byte) ([]byte, error) {
+
+			content = bytes.Replace(content,
+				[]byte("You may specify one optional `incident_urgency_rule` block configuring what urgencies to "+
+					"use.\nYour PagerDuty account must have the `urgencies` ability to assign an incident urgency "+
+					"rule.\nThe block contains the following arguments:"),
+				[]byte("The `incident_urgency_rule` block contains the following arguments:"),
+				1)
```

Comment:
Why not replace all here?

Reply:
I find it unintuitive to use ReplaceAll when there's only one incidence of the text. 🤷‍♀️